docs/patterns/actions/action-guide-without-agents.mdx

```diff
@@ -9,6 +9,58 @@ description: "Perform Actions with your account or on behalf of your users witho
 
 Composio lets you perform actions with your account or on behalf of your users. Feel free to check all the [tools](https://app.composio.dev/apps) we support. 
 
+### Get Parameters for an Action
```

Comment:
The new "Get Parameters for an Action" section is a valuable addition. Consider adding:
1. A brief explanation of why and when users might need to get parameters for an action.
2. A text-based alternative or summary of the video content for accessibility and quick reference.

Reply:
Done